src/app/public/modules/split-view/split-view.component.scss

```diff
@@ -7,12 +7,15 @@
   display: flex;
   border: 1px solid $sky-border-color-neutral-medium;
   background: #fff;
+  max-height: 100%;
 
   .sky-split-view-workspace-flex-container {
     flex-grow: 1;
     flex-shrink: 99;
     overflow: auto;
+    position: relative;
```

Comment:
Why do we need a `position: relative;` here?

Reply:
We don't. I verified that things are fine without it. I think it may have been a style from an initial version.

Reply:
Removed